src/auth/access-group-provider/access-group-from-graphql-api-call.service.ts

```diff
@@ -27,14 +26,19 @@ export class AccessGroupFromGraphQLApiService extends AccessGroupService {
   ): Promise<string[]> {
     const userId = userPayload.userId as string;
     const query = this.graphqlTemplateQuery.replace("{{userId}}", userId);
-    const response = await this.callGraphQLApi(query);
-    const accessGroups = this.responseProcessor(response);
+    try {
+      const response = await this.callGraphQLApi(query);
+      const accessGroups = this.responseProcessor(response);
 
-    Logger.log(
-      "Access groups from graphql api call getAccessGroups: " +
-        accessGroups.join(","),
-    );
-    return accessGroups;
+      Logger.log(
```

Comment:
Which logger is used here?

Reply:
The logger is automatically the one that we set when the application start.
Jay clarify it in an in-person Q&A